evm/src/main/java/com/horizen/evm/interop/TraceParams.java

```diff
@@ -0,0 +1,33 @@
+package com.horizen.evm.interop;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TraceParams {
+    @JsonProperty("enableMemory")
```

Comment:
Imho, all the annotations in this class are not necessary, except for `@JsonIgnoreProperties(ignoreUnknown = true)`.